recipes/bareditor/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "bareditor" %}
+{% set version = "0.17" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "6dbf897ff03a75182c23c3367dc110bf676857df9548e4bca6b0b452fdf0f432" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - wxpython
+    - numba
+    - pillow
+  run:
+    - python
+    - wxpython
+    - numba
+    - pillow
+
+test:
+  imports:
+    - bareditor
+    - bareditor.bareditor
+    - bareditor.imgdata
+
+about:
+  home: https://github.com/434625142/bareditor
+  license: BSD 3-clause
+  license_family: BSD
+  license_file: ''
+  summary: interactive python image-processing plugin framework
+  description: Image process framework based on plugin like imagej, it is esay to glue with scipy.ndimage, scikit-image, opencv, simpleitk, mayavi...and any libraries based on numpy
```

Comment:
Is this the description for this project? see: https://github.com/Image-Py/imagepy